docs/development.md

```diff
@@ -72,13 +73,15 @@ following chapters in a certain order:
 4.  Install pre-commit hooks
 
     ```shell-session
-    pre-commit install
+    $ cd /path/to/paperless
```

Comment:
Maybe put a blanket statement at the start that commands are run from the root of the clone, with any specific directories called out in the steps below (like `src` or `src-ui`)